demos/inpage/src/WaxInPage.tsx

```diff
@@ -477,7 +478,9 @@ export default class WaxInPage {
         description,
         'is',
         hexLen(bytes),
-        'bytes:',
+        `bytes (${(Number(measureCalldataGas(bytes)) / 16).toFixed(
+          1,
+        )} effective):`,
```

Comment:
What does effective mean here?

Reply:
I've been using it to mean equivalent non-zero bytes. So 4 zero-bytes equals one non-zero bytes since L1 charges 16 gas for non-zeros and 4 gas for zeros.